src/TLuaInterpreter.cpp

```diff
@@ -3547,6 +3565,47 @@ int TLuaInterpreter::createMapper(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#createVideoPlayer
+int TLuaInterpreter::createVideoPlayer(lua_State* L)
+{
+    int n = lua_gettop(L);
+    QString windowName = "";
```

Comment:
No need to initialise a `QString` (unless you need to differentiate between `QString::isNull()` and `QString::isEmpty()` - and very few cases do need to)...
```suggestion
    QString windowName;
```